libs/newsletter/testing/src/drivers/in-memory/newsletter.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+import { DaffNewsletterServiceInterface, DaffNewsletterSubmission } from '@daffodil/newsletter';
+import { DaffNewsletterUnion } from 'libs/newsletter/src/models/newsletter-union';
+
+/**
+ * The newsletter inmemory driver to mock the newsletter backend service.
+ * 
+ * @Param HttpClient
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffInMemoryNewsletterService implements DaffNewsletterServiceInterface<DaffNewsletterUnion, DaffNewsletterUnion>{ 
+  constructor() {}
+
+  send(email: DaffNewsletterUnion): Observable<DaffNewsletterUnion> {
+    return of(email);
```

Comment:
Can we have a way that we can specifically cause it to fail for testing purposes?

Reply:
Okay, I tried to address this by having it throw an error with an undefined payload.

Reply:
That'll work.